src/components/alerts/dashboard/_alerts-dashboard.scss

```diff
@@ -0,0 +1,82 @@
+.empty {
```

Comment:
This class will apply to the whole project. Please use BEM or scss structure to apply it only in this case.

Reply:
deleted because i don't need it anymore